pyhf/__init__.py

```diff
@@ -153,8 +156,10 @@ def add_or_get_modifier(self, channel, sample, modifier_def):
 
 class hfpdf(object):
     def __init__(self, spec, **config_kwargs):
-        self.config = modelconfig.from_spec(spec,**config_kwargs)
         self.spec = spec
+        self.schema = config_kwargs.get('schema', pkg_resources.resource_filename(__name__,'../validation/spec.json'))
```

Comment:
not sure what the best practice here is. to me `pkg_resources.resource_filename('pyhf','validation/spec.json'))` looks a bit nicer

Reply:
Unfortunately, it'll do it with respect to `__file__` which means we need an extra `../` to get to `validation/`. Similarly, `__name__ == 'pyhf'` for all intents and purposes. I don't know whether to be explicit here or not.

Reply:
if you rebase to `master`  `pkg_resources.resource_filename('pyhf','validation/spec.json'))` should now work